freqtrade/utils.py

```diff
@@ -322,3 +322,37 @@ def start_list_markets(args: Dict[str, Any], pairs_only: bool = False) -> None:
                   args.get('list_pairs_print_json', False) or
                   args.get('print_csv', False)):
             print(f"{summary_str}.")
+
+
+def start_test_pairlist(args: Dict[str, Any]) -> None:
+    """
+    Test Pairlist configuration
+    """
+    from freqtrade.pairlist.pairlistmanager import PairListManager
+    config = setup_utils_configuration(args, RunMode.UTIL_EXCHANGE)
+
+    exchange = ExchangeResolver(config['exchange']['name'], config, validate=False).exchange
+
+    quote_currencies = args.get('quote_currencies')
+    if not quote_currencies:
+        quote_currencies = [config.get('stake_currency')]
+    results = {}
+    for curr in quote_currencies:
+        config['stake_currency'] = curr
+        # Do not use ticker_interval set in the config
+        pairlists = PairListManager(exchange, config)
+        pairlists.refresh_pairlist()
+        results[curr] = pairlists.whitelist
+
+    for curr, pairlist in results.items():
+        print(f"Pairs for {curr}: ")
```

Comment:
This (human-readable) line should not be printed for `-1` format, it's machine-readable for easy iteration through the pairs.
I also not sure for `--print-json`, maybe this line should not also be printed in this case...

Reply:
i'm not certain about this ... 
yes, it should be machine readable (both, actually) - but simply chaining pairs together without any separation between the quote currencies (assuming more than one is used) seems odd to me, too.

Reply:
it will not be easy to iterate through the results, the header line should be additionally filtered out.
I assume json format is more for copy-paste into the pairlist file by a human, so it's probably not necessary for `--print-json` format; `-1` is more for script automation, the header will add the need to filter out this line in the scripts, instead of simple iterating through the result in the output.

Reply:
well i'm not sure - i find the -1 format very difficult to use (also for the other commands) - even in scripts.

using the json output and using `jq` seems simpler... 

I'll think of something.

Reply:
```
for p in `feqtrade test-pairlist -1`; do echo "do something useful with pair $p"; done
```

As I said above, for json it's questionable, so let's leave it as it is.